### PR TITLE
Virtualenv

### PR DESCRIPTION
**Pull request by adam-schwab**

### CODE REVIEW

**Comment by xunzhang** [comment]
Good work! I will test it in AWS soon

**Comment by xunzhang** [comment]
Can you include the cuda installation change section as well?

File: User_Installation.md [hunk 1]
```diff
@@ -27,34 +27,48 @@ export LD_LIBRARY_PATH="/usr/local/cuda/lib64"
...
 
-# Ubuntu 14.04 64-bit
+# Ubuntu 14.04 and 16.04 64-bit
 sudo apt-get update
-sudo apt-get install python-pip python-numpy python-paramiko
-
-# Ubuntu 16.04 64-bit
-sudo apt-get update
-sudo apt install python-pip python-numpy python-paramiko
+sudo apt-get install python-pip
```
Comment (xunzhang) [inline User_Installation.md]: you need to install paramiko before using psd_run, here you just deleted it.

File: User_Installation.md [hunk 2]
```diff
@@ -27,34 +27,48 @@ export LD_LIBRARY_PATH="/usr/local/cuda/lib64"
...
 
 # OSX
 brew install swig python homebrew/python/numpy
 sudo pip install wheel paramiko
 brew install git
 ```
 
-- Install protobuf
+#### OPTIONAL: Set up virtualenv
```
Comment (xunzhang) [inline User_Installation.md:52]: Can you also mention if users have tensorflow installed in their environment, it is required to use virtualenv.

**Review by xunzhang — COMMENTED** [review]
Also, how can I use `psd_run` in cluster environment?

File: User_Installation.md (hunk 1 above)
Comment (adam-schwab) [reply User_Installation.md]: Done

**Pull request merged**